docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -12,13 +12,15 @@ and [silverstripe/recipe-cms](https://github.com/silverstripe/recipe-cms)).
 
 ## Release Planning
 
-Releases are planned by assigning issues to GitHub milestones,
-according to our [roadmap](http://silverstripe.org/roadmap).
-SilverStripe is split up into many modules listed on [github.com/silverstripe](https://github.com/silverstripe).
+The Silverstripe CMS core team uses an Agile methodology. We aim to keep the latest development branches of each 
+module in a "potentially releasable" state. Releases are planned around an issue backlog according to our 
+[roadmap](https://www.silverstripe.org/software/roadmap/).  We re-revaluate our priorities every few weeks.
+
+Silverstripe CMS is split up into many modules listed on [github.com/silverstripe](https://github.com/silverstripe).
 Each of them will have its own milestones and issues, and different release lines (e.g. 1.x vs. 4.x).
```

Comment:
```suggestion
Each may have a different release line (e.g. 1.x vs. 4.x).
```

As per your comment above, referring to milestones on a module basis is less relevant now.